core-api-library/library/src/main/java/net/gini/android/core/api/authorization/UserRemoteSource.kt

```diff
@@ -0,0 +1,59 @@
+package net.gini.android.core.api.authorization
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import net.gini.android.core.api.RetrofitHelper
+import net.gini.android.core.api.authorization.apimodels.SessionResponseModel
+import net.gini.android.core.api.authorization.apimodels.UserRequestModel
+import net.gini.android.core.api.authorization.apimodels.UserResponseModel
+import net.gini.android.core.api.requests.SafeApiRequest
+import okhttp3.ResponseBody
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.PUT
+import retrofit2.http.Path
+
+class UserRemoteSource {
```

Comment:
Please pass in the UserService instance and the coroutine context via constructor.